snuba/datasets/processors/search_issues_processor.py

```diff
@@ -167,6 +168,15 @@ def _process_sdk_data(
         processed["sdk_name"] = _unicodify(sdk.get("name"))
         processed["sdk_version"] = _unicodify(sdk.get("version"))
 
+    def _process_contexts(
```

Comment:
We're thinking about having the 'custom' context processing done upstream within sentry app code to avoid doing business logicy stuff in this processor. 
Some examples:
https://github.com/getsentry/snuba/blob/8cf41e25b0fc2f97a84ddca14ad7e569d0d6314d/snuba/datasets/processors/transactions_processor.py#L256-L260
https://github.com/getsentry/snuba/blob/0c8a30814f441c491d9db76aab59b54fdc372b6a/snuba/datasets/processors/errors_processor.py#L170-L173

So, ideally, the incoming kafka message context should be written to the row with little to no custom processing.